src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -55,17 +55,33 @@ public void storeMessage(Message message) {
    * @return a list of messages posted by the user, or empty list if user has never posted a
    *     message. List is sorted by time descending.
    */
-  public List<Message> getMessages(String user) {
+
+  //Returns messages where the user is the recipient instead of the author.
```

Comment:
The documentation for this `getMessages` method is the part above starting with `/**` and ending with `*/` - you should edit that instead of starting a new comment.